cmake/CMakeLists.txt

```diff
@@ -213,7 +213,7 @@ else()
   set_target_properties(libprotobuf-lite PROPERTIES FOLDER "External/Protobuf")
   set_target_properties(libprotoc PROPERTIES FOLDER "External/Protobuf")
   set_target_properties(protoc PROPERTIES FOLDER "External/Protobuf")
-  add_library(protobuf::libprotobuf ALIAS libprotobuf)
+  add_library(protobuf::libprotobuf-lite ALIAS libprotobuf)
```

Comment:
Sounds wired.

https://cmake.org/cmake/help/latest/command/add_library.html#id5

Reply:
See comment above.

Reply:
The doc says:

add_library(<name> ALIAS <target>)

such that <name> can be used to refer to <target> in subsequent commands.

Reply:
Hmm, don't quite understand this as the code is indeed linking with the right library as intended or else I won't see so many build errors and reduction in size.